.github/workflows/watch.yml

```diff
@@ -1,6 +1,9 @@
 name: Watch for changes
 
 on:
+  push:
+    branches:
+    - bot-fork  # do not forget to revert
```

Comment:
This will only work on your local fork. Do you have all secrets there? 

Or should you push to async-aws/aws:bot-fork instead?

Reply:
> Or should you push to async-aws/aws:bot-fork instead
This is what I do